llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -14,24 +15,31 @@
 using namespace llvm;
 
 namespace {
-// Test fixture needed that holds the veariables needed by the parser.
+/// Perform tests against VFABI Rules. `invokeParser` creates a VFInfo object
+/// and a scalar FunctionType, which are used by tests to check that:
+/// 1. The scalar and vector names are correct.
+/// 2. The parameter number from the parsed mangled name matches the scalar ones
+///    given by the string FunctionType.
+/// 3. The number of vector parameters and their types match the values
+///    specified in the test.
+///    On masked functions it also checks that the last parameter is a mask (ie,
+///    GlobalPredicate).
+/// 4. The function is correctly found to have a mask.
```

Comment:
nit: The vector function

(since sometimes we mention here scalar functions as well)